### DISCUSSION TITLE
Outlier vs Anomaly vs Novelty

### DISCUSSION THREAD

**Discussion by rootonchair** [mlbvn/handson-ml2-vn#77] [Terminology] [answered]
Hiện `Anomaly` đang được dịch là `bất thường`. Còn `Outlier` và `Novelty` đang được dịch chung là `ngoại lai`

cc: @mlbvn/handson-ml2

**Comment by minhduc0711** [comment, marked as answer]
`outlier` và `novelty` có vẻ hơi khác nhau https://scikit-learn.org/stable/modules/outlier_detection.html
Dịch `novelty` thành `điểm mới lạ` hay gì đó tương tự được không?

Update: chốt lại là
* `anomaly` -> `bất thường`
* `novelty` -> `tính mới` / `điểm mới` (tùy context)

**Reply by cuongvng** [reply]
em vote "phát hiện điểm mới"

**Reply by lkhphuc** [reply]
`mới` không thể hiện hết ý nghĩa của `novelty` mà chỉ như `new` nhỉ? Mình vẫn thích `mới lạ` hơn.

**Reply by ngcthuong** [reply]
Novelty mÌnh thấy chỉ là "tính mới" thôi. Không cần "lạ". Đấy là nghĩa chung chung của từ novelty, đặc biệt trong nghiên cứu viết báo.

**Reply by minhduc0711** [reply]
Trong sách thì từ này chỉ đi theo cụm `novelty detection`, nên mình thấy dịch là `phát hiện tính mới` ok.
